src/Nri/Ui/Tabs/V5.elm

```diff
@@ -0,0 +1,341 @@
+module Nri.Ui.Tabs.V5 exposing
+    ( view
+    , Alignment(..)
+    , Tab, viewTabDefault
+    )
+
+{-|
+
+@docs view
+@docs Alignment
+@docs Tab, viewTabDefault
+
+-}
+
+import Accessibility.Styled.Aria as Aria
+import Accessibility.Styled.Role as Role
+import Accessibility.Styled.Widget as Widget
+import Css exposing (..)
+import EventExtras
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes as Attributes
+import Html.Styled.Events as Events
+import Json.Decode
+import List.Zipper exposing (Zipper)
+import List.Zipper.Extra
+import Nri.Ui
+import Nri.Ui.Colors.Extra
+import Nri.Ui.Colors.V1 as Colors
+import Nri.Ui.Fonts.V1
+import Nri.Ui.Html.Attributes.V2 as AttributesExtra
+
+
+{-| Determines whether tabs are centered or floating to the left or right.
+-}
+type Alignment
+    = Left
+    | Center
+    | Right
+
+
+{-| -}
+type alias Tab id msg =
+    { id : id
+    , idString : String
+    , tabView : Html msg
+    , panelView : Html msg
+    , spaHref : Maybe String
+    }
+
+
+{-| -}
+view :
+    { title : Maybe String
+    , alignment : Alignment
+    , customSpacing : Maybe Float
+    , onSelect : id -> msg
+    , onFocus : String -> msg
+    , selected : id
+    , tabs : List (Tab id msg)
+    }
```

Comment:
it's starting to feel like there's enough config here to move to an HTML-like pattern.

Reply:
I'm not sure -- there are 4 mandatory values: `onSelect`, `onFocus`, `selected`, and `tabs`.

That leaves `alignment`, `title`, and `customSpacing`. I'm pretty sure that `title` should only be present when the `alignment` is `Right`... I could set a default for the two remaining properties, `alignment: Left` and `customSpacing: Nothing` but I'm not sure that at this point, that's better than the config-based API.